src/components/base/Footer.vue

```diff
@@ -17,17 +17,19 @@
         </ul>
       </div>
       <div class="col-sm-12 col-md-4 col-lg-4">
-        <h5 class="mb-3">Follow me</h5>   
-        <a target="_blank" href="https://web.facebook.com/fredrickot/"><font-awesome-icon :icon="{ prefix: 'fab', iconName: 'facebook' }" class="mr-3 facebook" /></a>
-        <a target="_blank" href="https://twitter.com/home"><font-awesome-icon :icon="{ prefix: 'fab', iconName: 'twitter' }" class="mr-3 twitter" /></a>
-        <a target="_blank" href="https://www.linkedin.com/in/otim-fredrick-29730a86/"><font-awesome-icon :icon="{ prefix: 'fab', iconName: 'linkedin-in' }" class="mr-3 linkedin-in" /></a>
-        <a target="_blank" href="https://github.com/otim22"><font-awesome-icon :icon="{ prefix: 'fab', iconName: 'github' }" class="mr-3 github" /></a>
-        <a target="_blank" href="https://medium.com/@otimfredrick"><font-awesome-icon :icon="{ prefix: 'fab', iconName: 'medium' }" class="medium" /></a>
+        <h5 class="mb-3">Follow me</h5> 
+        <div class="mt-4">
+          <a target="_blank" href="https://web.facebook.com/fredrickot/"><font-awesome-icon :icon="{ prefix: 'fab', iconName: 'facebook' }" class="mr-3 facebook" /></a>
```

Comment:
How about we create a component wrapper to handle the social media specs.

 **DRY**

We can create a helper component to group this data. E.g `<social-media-list/>`


```
<template>
    <a target="_blank" :href="url">
        <font-awesome-icon :icon="{ prefix, iconName }" :class="bindClass" />
    </a>
</template
<scrpit>
......
props: {
    url: {
        type: string
        required: true //https://web.facebook.com/fredrickot
    },
    prefix: {
        type: String,
        required: true // fab
    },
    iconName: {
        type: String,
        required: true // facebook
    },
    className: {
        type: String,
        required: true // mr-3
    }
}
.....
computed: {
    bindClass () {
        return `${className} ${iconName}`
    }
}
</scprit>
```

We make use of `v-for` by creating a list of objects from the component `props`